packages/terra-cell-grid/src/Cell.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Cell.module.scss';
+import CellGridUtils from './_CellGridUtils';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content to be displayed for the row cell
+   */
+  children: PropTypes.node,
+  /**
+   * Function callback for the ref of the td.
+   */
+  refCallback: PropTypes.func,
+  /**
+   * Function callback for the ref of the td.
+   */
+  width: PropTypes.shape({
+    static: PropTypes.shape({
+      value: PropTypes.number,
+      unit: PropTypes.string,
+    }),
+    percentage: PropTypes.number,
+    scalar: PropTypes.number,
+  }),
+};
+
+const defaultProps = {
+  children: [],
```

Comment:
Should we provide a default? If children are null, should we render nothing?

Reply:
I vote nay

Reply:
![](https://media.giphy.com/media/3Hm4TlhPjZu3C/giphy.gif)